README.md

```diff
@@ -368,294 +396,260 @@ A device renderer instance can be configured using the `options` argument (objec
 
-### `microphone`
+```html
+<!-- OPTIONAL: Import google maps library with your API key to enable map positioning feature -->
+<script src="https://maps.googleapis.com/maps/api/js?key=xxxxxxxxxxxxxxxxxxxxxxxx-yyyyyyyyyyyyyy"></script>
+```
 
-<img align="right" src="./doc/assets/ic_camera_mic_active_black.svg" alt="..."></img>
+#### `gpsSpeedSupport`
 
 -   **Type:** `Boolean`
 -   **Default:** `false`
--   **Compatibility:** `PaaS`
+-   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables microphone injection. This can be used to forward local microphone (or webcam audio) to the Android virtual device.
-
-### `fileUpload`
+    Enables or disables GPS speed support.
 
-<img align="right" src="./doc/assets/ic_cloud_upload_active_black.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_id.svg"> `identifiers`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Identifiers`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the fileUpload widget and drag & drop. This widget can be used to forward local file to the Android virtual device. When drag & dropping APK or ZIP files, it will install them.
+    Enables or disables the identifiers widget. This widget can be used to set the identifiers (Android ID / IMEI) of the Android virtual device.
 
-### `fileUploadUrl`
+#### `keyboard`
 
--   **Type:** `String`
--   **Default:** `undefined`
+-   **Type:** `Boolean`
+-   **Default:** `true`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Set the file upload url, required if `fileUpload` is set to `true`.
+    Enables or disables the keyboard widget. This widget can be used to transmit keyboard keystrokes to the Android virtual device.
 
-### `clipboard`
-
-<img align="right" src="./doc/assets/ic_clipboard.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_keymapping.svg"> `keyboardMapping`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `KeyboardMapping`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the clipboard widget. This widget can be used to forward local clipboard to the Android virtual device.
+    Enables or disables the keyboard mapping. This widget can be used to map keys with commands (e.g., tap, swipe-left, tilt, etc.).
 
-### `battery`
-
-<img align="right" src="./doc/assets/ic_battery.svg" alt="..."></img>
+#### `microphone`
 
 -   **Type:** `Boolean`
--   **Default:** `true`
--   **Compatibility:** `PaaS`, `SaaS`
+-   **Default:** `false`
+-   **Compatibility:** `PaaS`
 -   **Details:**
-    Enables or disables the battery widget. This widget can be used to set the battery level and state of the Android virtual device.
-
-### `gps`
+    Enables or disables microphone injection. This can be used to forward local microphone (or webcam audio) to the Android virtual device.
```

Comment:
```suggestion
    Enables or disables microphone injection. This can be used to forward the local microphone (or webcam audio) to the Android virtual device.
```